### PR TITLE
Only allow custom MWI when running locally

### PR DESCRIPTION
**Pull request by delosh653**
Update tool to only allow custom MWI when running locally. Set to `FALSE` by default.

Also fixes some about/documentation typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +36,53 @@ The MWI creates a score for each zip code between 0 and 100, such that:
 
 The MWI was developed with the mental health status of Black Americans in mind. We selected Black Americans as a priority population in order to center at the margins and avoid creating an index that is focused to the 'average community.'  We believe that focusing on Black Americans in this way allows all groups experiencing disparities to benefit because none of us are well until all of us are well. We also recognize the need to identify, recognize, and adapt the MWI for additional priority populations.
 
+# Instructions to Create Your Own MWI
+
+To create your own Mental Wellness Index, you must be running the Mental Wellness Index Tool on your local computer in order to protect your data. Follow the instructions below to create your own MWI for your community below by adjusting weights and/or adding your own data and metadata.
+
+1. Download R and RStudio. Download a modern browser (Firefox, Chrome, Edge, etc.) and make that your default browser if you haven't already.
```
Comment (karenmjiang) [inline README.md:43]: Add hyperlink to R and RStudio? 

https://www.rstudio.com/products/rstudio/download/
https://www.r-project.org/

May also want to mention somewhere that all of this is free.

File: README.md [hunk 2]
```diff
@@ -26,6 +36,53 @@ The MWI creates a score for each zip code between 0 and 100, such that:
...
+7. In the top right hand corner of the "app.R" window, you should see "Run App". Click the small downard arrow to the right of that and click "Run External". Then click "Run App".
+
+8. After a delay (this will be slow the first time, then quicker after that), the Mental Wellness Index Tool should open in your browser. Click on the "Create Your Own MWI" tab and follow the remaining steps to create your own MWI.
+
+9. If you are only adjusting weights for included data, skip the next step.
+
+10. Put each of your datasets in a CSV (comma separated value) format, with one column corresponding to the geographical ID of the data, a column corresponding to the numerator of the data, and another column corresponding to the denominator (if needed).
+   * Accepted geographical ID types are always numeric and include the following:
+      * ZCTA (example: 35406)
+      * County (example: 01001)
```
Comment (karenmjiang) [inline README.md:65]: * 5 digit ZIP Code (US Postal Service)
* 5 digit ZCTA Code
* 5 digit County FIPs Code (2 digit state + 3 digit county)
* 11 digit Tract Code (2 digit state + 3 digit county + 6 digit tract)

File: README.md [hunk 3]
```diff
@@ -26,6 +36,53 @@ The MWI creates a score for each zip code between 0 and 100, such that:
...
+   * If a denominator column is provided, the final input to the MWI will be the numerator divided by the denominator, multiplied by the scaling number (specified in the metadata file, see next step).
+   * Numerators and denominators must be numeric columns.
+   * Missing data should have cells left blank.
+   * If race stratified, there should be two columns: one ending in '_pop' corresponding to the overall population measure, and one ending in '_black' corresponding to the black populations measure. In the Metadata.xlsx file edit, that row's 'Preprocessed' column should be set to TRUE.
+
+11. Download Metadata.xlsx with the button below. If adding custom data, add a row and fill in information for each measure you want to add to the Mental Wellness Index. Descriptions for each column can be found in the 'Column Descriptions' sheet of the Metadata.xlsx. Note that all column names, with the exception of 'denominator', must be filled out.
+   * If you have multiple measures in one file, add a row for each measure and its qualities, but specify the same file name.
+   * If you would not like to include a measure in your MWI, either delete the measure row or set its weight to 0.
```
Comment (karenmjiang) [inline README.md:75]: If you would like to remove an existing measure in your MWI, either delete the measure row or set it weight to 0.

File: app.R [hunk 4]
```diff
@@ -885,15 +910,41 @@ ui <- fluidPage(
...
+                "<br>",
+                "<li> In the console window, which is in the bottom left hand corner, enter the following line and answer \"yes\" to all prompts in the console as you install these packages:</li>",
+                "<ul>",
+                "<li>install.packages('readxl', 'writexl', 'htmltools', 'shiny', 'tigris', 'leaflet', 'RColorBrewer', 'sf', 'plotly', 'ggbeeswarm', 'shinyWidgets', 'sass', 'shinycssloaders', 'shinyBS')</li>",
+                "</ul>",
+                "<br>",
+                "<li> In \"app.R\", navigate to line 11, which should say \"app_local <- FALSE\". Change FALSE to TRUE.</li>",
+                "<br>",
+                "<li> In the top right hand corner of the \"app.R\" window, you should see \"Run App\". Click the small downard arrow to the right of that and click \"Run External\". Then click \"Run App\".</li>",
+                "<br>",
```
Comment (karenmjiang) [inline app.R:939]: typo line 938: downward

**Review by karenmjiang — APPROVED** [review]
Looks good, added a few comments in the README section, and typo correction in the app.R.

File: app.R (hunk 4 above)
Comment (delosh653) [reply app.R:939]: Fixed!

File: README.md (hunk 3 above)
Comment (delosh653) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (delosh653) [reply README.md]: Added these general clarifications.

File: README.md (hunk 1 above)
Comment (delosh653) [reply README.md]: Whoops! Had these in the app, but not the README. Added these. Also added the free bit.

**Pull request merged**